src/persistence.js

```diff
@@ -882,11 +882,11 @@ export default function _Persistence(mpInstance) {
         return '';
     };
 
-    this.getAllUserAttributes = function(mpid) {
+    this.getUserIdentities = function(mpid) {
```

Comment:
it looks like this has been added back in after removing it from https://github.com/mParticle/mparticle-web-sdk/pull/869/files.  probably a rebasing issue.